torq.q

```diff
@@ -610,14 +611,18 @@ if[@[value;`.ps.loaded;0b]; .ps.initialise[]]
 // initialise connections
 if[@[value;`.servers.STARTUP;0b]; .servers.startup[]]
 
+.proc.run:{[id;a]
+    .lg.o[id;"attemping to run: ",.Q.s1 a];
```

Comment:
think the style guideline is 2 spaces of indentation within functions (same in `.proc.init`)

Reply:
@ThomasSmyth what's the ruling here in regards to maintaining consistency?